packages/big-design/src/components/Select/Select.tsx

```diff
@@ -110,7 +110,7 @@ export const Select = typedMemo(
     const handleOnHighlightedIndexChange = (
       changes: Partial<UseComboboxState<SelectOption<T> | SelectAction | null>>,
     ) => {
-      if (typeof changes.highlightedIndex !== 'undefined' && changes.highlightedIndex > -1) {
+      if (typeof changes.highlightedIndex !== 'undefined') {
```

Comment:
Do we also need this fix in Dropdown / Multi?

Reply:
Nope, only relevant to Select.